extlinks/programs/tests.py

```diff
@@ -193,6 +194,23 @@ def test_program_detail_user_list_form(self):
         self.assertEqual(response.context_data['total_added'], 1)
         self.assertEqual(response.context_data['total_removed'], 0)
 
+    def test_program_detail_namespace_form(self):
+        """
+        Test that the date limiting form works on the program detail page.
```

Comment:
The docstring here looks to be exactly copied from the above test. Could you please change "date" to "namespace id"?

Reply:
Fixed that, and the one above that which was also exactly copied from the one above that.

Reply:
Thanks! @Samwalton9 will give a final review and approve.

Reply:
@Samwalton9, @Jain-Aditya: Please don't forget to also update the GCI task itself, which means you need to click a buton (either Approved or More Work Needed) until midnight UTC. Thanks! :)

Reply:
Sure @aklapper! We will update the GCI task. Thanks :)